resources/definitions/fdmextruder.def.json

```
@@ -62,6 +62,18 @@
                     "settable_per_meshgroup": false,
                     "type": "str"
                 },
+                "machine_extruder_end_code_duration":
+                {
+                    "default_value": 0,
+                    "description": "Duration it takes to execute the end g-code when switching away from this extruder.",
```

```suggestion
                    "description": "The time it takes to execute the end g-code, when switching away from this extruder.",
```

English lesson time: It's either 'the time it takes' or 'the duration of', not 'duration it takes'.